blocks/library/verse/style.scss

```diff
@@ -0,0 +1,8 @@
+pre.wp-block-verse,
+.wp-block-verse pre {
+	white-space: nowrap;
+	font-family: inherit;
+	font-size: inherit;
```

Comment:
I am not sure, but some way to differentiate besides margins could be good. Slightly smaller font could give more room for lines to not wrap.